oars/models/base/dim_date.sql

```diff
@@ -0,0 +1,15 @@
+with days as (
+  {{ dbt_utils.date_spine(
+    datepart='day',
+    start_date="date('2018-01-01')",
```

Comment:
I think we actually need to go back to 2013 here, those are the earliest enrollments we might encounter